extensions/distributed-tracing.md

```diff
@@ -1,55 +1,43 @@
 # Distributed Tracing extension
 
 This extension embeds context from
-[Distributed Tracing](https://w3c.github.io/trace-context/) so that distributed
-systems can include traces that span an event-driven system. This extension is
-meant to contain historical data of the parent trace, in order to diagnose
-eventual failures of the system through tracing platforms like Jaeger, Zipkin,
-etc.
+[W3C TraceContext](https://www.w3.org/TR/trace-context/) into a CloudEvent.
+
+The [OpenTelemetry specification](
```

Comment:
You could introduce OTel in one sentence with a link to https://opentelemetry.io.

Reply:
I changed it a bit, to try to say how this extension related to OTel and what OTel is.